plugins/thumbnail/lib/model/thumbnailFactory/kBaseResizeAdapter.php

```diff
@@ -213,6 +219,53 @@ public function resizeEntryImage($params)
 		return $this->finalThumbPath;
 	}
 
+	protected function initOrigImagePath()
+	{
+		$entry = $this->getEntry();
+		/* @var  $fileSync FileSync*/
+		$fileSync = $this->parameters->get(kThumbFactoryFieldName::FILE_SYNC);
+		if ($fileSync)
+		{
+			$orig_image_path = $fileSync->getFullPath();
+		}
+		else
+		{
+			$orig_image_path = $this->parameters->get(kThumbFactoryFieldName::ORIG_IMAGE_PATH);
+		}
+
+		if ($orig_image_path === null || !kFile::checkFileExists($orig_image_path))
```

Comment:
orig_image_path can be also empty string, it is better to check empty($orig_image_path)